tabular/src/autogluon/tabular/learner/default_learner.py

```diff
@@ -144,7 +144,7 @@ def _fit(
             predict_n_time_per_row = trainer.get_model_attribute_full(model=trainer.model_best, attribute="predict_n_time_per_row")
             predict_n_size = trainer.get_model_attribute_full(model=trainer.model_best, attribute="predict_n_size", func=min)
             if predict_n_time_per_row is not None and predict_n_size is not None:
-                log_throughput = f" | Estimated inference throughput: {(1/predict_n_time_per_row):.1f} rows/s ({int(predict_n_size)} batch size)"
+                log_throughput = f" | Estimated inference throughput: {1/(predict_n_time_per_row if predict_n_time_per_row else np.finfo(np.float16).eps):.1f} rows/s ({int(predict_n_size)} batch size)"
```

Comment:
Very nice catch! Can you also add the fix to AbstractTrainer? https://github.com/autogluon/autogluon/blob/78a10468ab51121c5d16175c5df0ca582168da80/core/src/autogluon/core/trainer/abstract_trainer.py#L2179-L2182

Reply:
Ah, I just noticed that this part avoids the div by zero via the if check. I will unify it. I think logging is better than not logging.